src/app/static/src/tests/index.test.js

```diff
@@ -66,10 +58,108 @@ describe("index page", () => {
         '        </thead>\n' +
         '    </table>';
 
-    require("../js/index");
+    const reports = [{
```

Comment:
Rather than adding selenium tests have tested with jsdom. Of course this relies on the above template being correct, but we do also have template tests, so all the units are under test. It would be even nicer if down the line we could combine our template and js tests somehow, maybe one to look into!